src/graphql/admin/index.ts

```diff
@@ -13,7 +13,7 @@ export const gqlAdminSchema = new GraphQLSchema({
   types: ERROR_INTERFACE_TYPES,
 })
 
-if (isDev) {
+if (!isProd) {
```

Comment:
will this work in test environment? do we have permissions to write?

Reply:
Testnet environment has `NODE_ENV=production` set so as to be as close to production as possible.